src/components/cards/skia-cards/KingOfTheHillCard.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useCallback, useMemo } from 'react';
+import * as i18n from '@/languages';
+import { useDimensions } from '@/hooks';
+import { SkiaCard } from './SkiaCard';
+import { Blur, Group, Image, Paint, useImage } from '@shopify/react-native-skia';
+import { AnimatedText, Box, ColorModeProvider, globalColors, Inline, Text, TextShadow } from '@/design-system';
+import { ShinyCoinIcon } from '@/components/coin-icon/ShinyCoinIcon';
+import { formatCurrency, formatNumber } from '@/helpers/strings';
+import { getSizedImageUrl } from '@/handlers/imgix';
+import { useNavigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import { useDerivedValue } from 'react-native-reanimated';
+import { useAnimatedCountdown } from '@/hooks/reanimated/useAnimatedCountdown';
+import { KingOfTheHillKing } from '@/state/kingOfTheHill/kingOfTheHillStore';
+
+const CARD_HEIGHT = 84;
+const startingNextRoundText = i18n.t(i18n.l.king_of_hill.starting_next_round);
+const leftText = i18n.t(i18n.l.king_of_hill.left);
+
+function AnimatedCountdownText({ targetUnixTimestamp, color }: { targetUnixTimestamp: number; color: string }) {
+  const countdown = useAnimatedCountdown(targetUnixTimestamp);
+
+  const timeString = useDerivedValue(() => {
+    if (countdown.value === '00:00:00') return startingNextRoundText;
+    return `${countdown.value} ${leftText}`;
+  });
+
+  return (
+    <TextShadow blur={8} shadowOpacity={0.24} color={color}>
+      <AnimatedText color={{ custom: color }} tabularNumbers size="11pt" weight="heavy">
+        {timeString}
+      </AnimatedText>
+    </TextShadow>
+  );
+}
+
+export function KingOfTheHillCard({ king }: { king: KingOfTheHillKing }) {
+  const { navigate } = useNavigation();
+  const { token } = king;
+  const coinIconImage = useImage(king.token.iconUrl);
+
+  const { width } = useDimensions();
+  const cardWidth = width - 20 * 2;
+  const hasTokenPriceIncreased = token.price.relativeChange24h > 0;
```

Comment:
will we always have `price.relativeChange24h`?  just making sure, other parts of the app we optionally check this:
```
(token?.price?.relativeChange24h ?? 0)
```